vars/apache.yml

```diff
@@ -173,17 +173,18 @@ apache_vhosts:
       # URLs with no language code need to be assigned one by wsgi.py
       WSGIScriptAliasMatch ^/$ /var/www/html/start/wsgi.py
       WSGIScriptAliasMatch ^/about[/]?$ /var/www/html/start/wsgi.py
+      WSGIScriptAliasMatch ^/contact /var/www/html/start/wsgi.py
       WSGIScriptAliasMatch ^/donate /var/www/html/start/wsgi.py
       WSGIScriptAliasMatch ^/download /var/www/html/start/wsgi.py
-      WSGIScriptAliasMatch ^/features[/]?$ /var/www/html/start/wsgi.py
       WSGIScriptAliasMatch ^/newsletter /var/www/html/start/wsgi.py
-      WSGIScriptAliasMatch ^/organizations[/]?$ /var/www/html/start/wsgi.py
```

Comment:
hm, I forget what we decided to do with this page, but I don't think we should simply delete it. I'm sure there are links to this on various mailing lists and in company docs and who knows where.

Maybe send them to https://wiki.mozilla.org/Thunderbird/Enterprise for now.

Reply:
I dont think that went anywhere aside from the corporate supporter section on donate. The wiki link is probably a good idea!

Reply:
I'll update this with that fix tomorrow morning.